source/_integrations/venstar.markdown

```diff
@@ -46,6 +46,14 @@ The following values are supported for the preset_mode state attribute:
 - `temperature`: *Disables* the schedule and holds the set temperature indefinitely.
 - `away`: Places the thermostat in away mode
 
+The meanings of the values of the climate entity's `schedule_part` state attribute are:
```

Comment:
I notice the other sensor entities from this integration are not documented, and now that it is an enum sensor with descriptive names instead of opaque numbers, should this be documented at all?

Reply:
Yeah, a short mention in the list of supported features should be enough.